### ISSUE TITLE
Preventing swipe right/left on a specific tab

### ISSUE THREAD

**Issue by Danny-weebo** [react-native-community/react-native-tab-view#412]
Hey!
I'm trying to prevent swiping left on a specific tab (without canceling the option to swipe right)

### What have you tried
I tried doing it with `canJumpToTab` prop (returning false when the key name is the tab that I don't want it to be swipe-able to rich) but I don't like how it reacts - when I'm swiping to that component and then automatically after a few seconds it swipes back. (I'd prefer to block the option to swipe to that direction in the first place and not show a 'peek' from that component).

thanks!

**Comment by satya164** [comment]
Have you tried setting the `swipeEnabled` prop?

**Comment by Danny-weebo** [comment]
`swipeEnabled` prop isn't relevant as I want to prevent swipe on for one direction and not both ways.

**Comment by MrLoh** [comment]
I have the same use case, no idea about a solution though, not sure it's even easily possible to enable that by modifying the library.

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.

**Comment by stephanoparaskeva** [comment]
@satya164  Still having said issue, how can one disable swipe in 1 direction only? 

This is for a form, where the user can swipe to the next layout but only after they make their selection. But the user can always choose to go back (swipe left).

**Comment by grsoares21** [comment]
Hello. Has anyone found a solution to this?
I'd imagine we would have to use a custom pager to do this, but I have no idea where to begin.

**Comment by tudiantuan** [comment]
Hello. I have same problem. Has anyone found a solution to this?
